.github/workflows/compatibility.yml

```
@@ -12,7 +12,7 @@ jobs:
       max-parallel: 10
       matrix:
         # Minimum support version, ..., Latest version, `beta`.
-        woocommerce: [ '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.1', '6.2.2', '6.3.0', '6.3.1', 'beta' ]
+        woocommerce: [ '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.1', '6.2.2', '6.3.0', '6.3.1', '6.4.0', '6.4.1', '6.5.0', '6.5.1' 'beta' ]
```

The latest minor version for each major version is enough. 

```suggestion
        woocommerce: [ '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.2', '6.3.1', '6.4.1', '6.5.1' 'beta' ]
```

---

Fixed.